plugins/kapt3/kapt3-compiler/src/org/jetbrains/kotlin/kapt3/stubs/ClassFileToSourceStubConverter.kt

```diff
@@ -751,10 +751,24 @@ class ClassFileToSourceStubConverter(val kaptContext: KaptContextForStubGenerati
         val value = field.value
 
         val origin = kaptContext.origins[field]
+        // Static fields in interfaces needs an initializer, even when just processing it using APT
+        val isStaticInterfaceField = containingClass.isInterface() && isStatic(field.access)
```

Comment:
Is `isStatic(field.access)` redundant? All fields in interface should be static.

Reply:
True, but I figure it's better to be explicit.